engine/src/mblhandlers.cpp

```diff
@@ -1324,8 +1324,12 @@ Exec_stat MCHandleStartTrackingSensor(void *p_context, MCParameter *p_parameters
     if (p_parameters)
     {
         MCAutoValueRef t_value;
+        MCAutoBooleanRef t_bool;
         p_parameters->eval(ctxt, &t_value);
-        t_loosely = MCValueIsEqualTo(*t_value, kMCTrue);
+        // PM-2015-03-11: [[ Bug 14855 ]] Evaluate correctly the second param
+        /* UNCHECKED */ ctxt . ConvertToBoolean(*t_value, &t_bool);
```

Comment:
Could you rather throw an error if `ctxt . ConvertToBoolean` returns false? That causes a crash if the user passes a value which cannot be converted to a bool (such as an array).